profile/README.md

```diff
@@ -5,6 +5,7 @@ You may learn more about our tools and our work in progress by checking out the
 following links:
 
 - [The epiverse-trace R-universe](https://epiverse-trace.r-universe.dev/)
```

Comment:
Any opinion about removing the **r-universe link** from README too to avoid redundancies? It is already in the header of the new _dev corner_ site

Reply:
I do like having a link to r-universe there. It increases the visibility to the r-universe, which is the only page that provides an overview of the different packages we are building and what they're doing.